### PR TITLE
Consolidate governance documents in Vega Organization

### PR DESCRIPTION
**Pull request by domoritz**
Corresponding pull request in the Vega org: https://github.com/vega/.github/pull/2

### CODE REVIEW

**Review by jonmmease — APPROVED** [review]
Looks good!

**Comment by binste** [comment]
Requested @mattijn's approval as well as he was not on the call. @mattijn I summarised the preliminary decisions, pending your approval, in https://github.com/altair-viz/altair/discussions/3164#discussioncomment-7707609

**Review by binste — APPROVED** [review]
Looks good to me as well, thanks @domoritz !

**Review by joelostblom — APPROVED** [review]

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,132 +1,3 @@
...
-For answers to common questions about this code of conduct, see the FAQ at
-[https://www.contributor-covenant.org/faq][FAQ]. Translations are available at
-[https://www.contributor-covenant.org/translations][translations].
-
-[homepage]: https://www.contributor-covenant.org
-[v2.1]: https://www.contributor-covenant.org/version/2/1/code_of_conduct.html
-[Mozilla CoC]: https://github.com/mozilla/diversity
-[FAQ]: https://www.contributor-covenant.org/faq
-[translations]: https://www.contributor-covenant.org/translations
+We as a project of the Vega Organization use the [Vega Code of Conduct](https://github.com/vega/.github/blob/main/CODE_OF_CONDUCT.md).
```
Comment (arvind) [inline CODE_OF_CONDUCT.md:3]:
```suggestion
As a project of the Vega Organization, we use the [Vega Code of Conduct](https://github.com/vega/.github/blob/main/CODE_OF_CONDUCT.md).
```

**Comment by domoritz** [comment]
The governance docs are now at https://github.com/vega/.github. This can be merged.

**Comment by mattijn** [comment]
Thanks for the effort

**Pull request merged**